tools/config/seed.config.ts

```diff
@@ -50,26 +56,16 @@ export class SeedConfig {
 
   NG2LINT_RULES        = customRules();
 
-
-  // Declare NPM dependencies (Note that globs should not be injected).
-  DEV_NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencies([
+  NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencies([
     { src: 'systemjs/dist/system-polyfills.src.js', inject: 'shims' },
     { src: 'reflect-metadata/Reflect.js', inject: 'shims' },
     { src: 'es6-shim/es6-shim.js', inject: 'shims' },
     { src: 'systemjs/dist/system.src.js', inject: 'shims' },
     { src: 'angular2/bundles/angular2-polyfills.js', inject: 'shims' },
-    { src: 'rxjs/bundles/Rx.js', inject: 'libs' },
-    { src: 'angular2/bundles/angular2.js', inject: 'libs' },
-    { src: 'angular2/bundles/router.js', inject: 'libs' },
-    { src: 'angular2/bundles/http.js', inject: 'libs' }
-  ]);
-
-  PROD_NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencies([
```

Comment:
Normalizing here is redundant with the normalization done with `get DEPENDENCIES()`